lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -4,8 +4,10 @@ class MulticastLogger < Logger
 
     def initialize(*loggers)
       require 'set'
-      @loggers = Set.new(loggers)
-      @level   = DEBUG
+      @loggers           = Set.new(loggers)
+      @level             = DEBUG
+      @default_formatter = VMDBLogger::Formatter.new
```

Comment:
The alternative for this is we could call `super`, but incase that would break other things, we decided it would be better to start with this change for now, since it will modify less things in doing so.  Up for debate though.